README.md

```diff
@@ -102,7 +102,7 @@ If you want to contribute to the project, please see: https://opentimelineio.rea
 
 You can get the latest development version via:
 
-`git clone git@github.com:PixarAnimationStudios/OpenTimelineIO.git OpenTimelineIO`
+`git clone --recursive git@github.com:PixarAnimationStudios/OpenTimelineIO.git OpenTimelineIO`
```

Comment:
This isn't wrong, but also cmake should do the recursive init for you (unless that doesn't work on windows?)  This shouldn't be necessary in other words.

Reply:
I'll test that - I think you are correct that it works. I made this note at the point when the cmake files errored out and didn't run at all.

Reply:
removed --recursive